src/bodywork/cli/cli.py

```diff
@@ -545,3 +567,31 @@ def setup_namespace(args: Namespace) -> None:
     load_kubernetes_config()
     setup_namespace_with_service_accounts_and_roles(namespace)
     sys.exit(0)
+
+
+def validate_config(args: Namespace) -> None:
+    """Validates a Bodywork config file and returns errors.
+
+    :param args: Arguments passed to the run command from the CLI.
+    :param logger: Bodywork logger.
+    """
+    file_path = Path(args.file)
+    check_py_files = args.check_files
+    try:
+        BodyworkConfig(file_path, check_py_files)
+        print(f'--> {file_path} is a valid Bodywork config file.')
+        sys.exit(0)
+    except FileExistsError as e:
+        print(f'--> {e}')
+        sys.exit(1)
+    except BodyworkConfigParsingError as e:
+        print(f'--> {e}')
+        sys.exit(1)
+    except BodyworkConfigMissingSectionError as e:
+        print(f'--> {e}')
+        sys.exit(1)
```

Comment:
If these three exceptions are going to be treated the same, then just put them in one except block:

except ex1, ex2, ex3

Reply:
Yes 🙂